### DISCUSSION TITLE
Set the line height of Markdown cells?

### DISCUSSION THREAD

**Discussion by paul-rogers** [microsoft/vscode-jupyter#14306] [General]
How do I control the line height of Markdown cells in Jupyter in VS Code? The editor cell line height is fine. However, when I click the check mark to see the formatted text, the lines are very close to one another and quite hard to read. By contrast, regular VS Code (non-Jupyter) Markdown Preview line height is fine. Also, the same notebooks shown in (non-VS Code) Juypter are fine.

I'm using the latest VS Code and Jupyter extension, latest Linux Mint OS.

There is a `markdown.preview.lineHeight` setting in VS code, however that does not appear to apply to Jupyter.

Is there some other setting somewhere that will solve the issue?

**Comment by amunger** [comment]
I'm pretty sure that setting should also affect markdown in notebooks. created https://github.com/microsoft/vscode/issues/193857 to address this

**Comment by ddahan** [comment]
```
"notebook.markdown.lineHeight": 20
```

**Comment by paul-rogers** [comment]
When this issue was reported, the Markdown line height in Jupyter cells was independent of that in, say, the Markdown document preview. Someone made a fix to allow setting the Jupyter Markdown line height, but the setting is in pixels, which does not work well across screens or devices: one needs a different setting for each dot pitch. Also, it made multi-line titles unreadable.

A unified setting across all Markdown windows and views would be ideal. For now, the pixel setting at least partially solves the problem and continues to work for me in the latest version.
